significantDimension.js

```diff
@@ -0,0 +1,75 @@
+import tailLength from './tailLength';
+import round from './round';
+import { uniq } from 'underscore';
+
+/**
+ * computes the significant dimension for a list of numbers
+ * That's the number of decimals to which we can round the numbers
+ * without loosing information
+ *
+ * @exports significantDimension
+ * @kind function
+ *
+ * @example
+ * import {significantDimension} from '@datawrapper/shared/significantDimension';
+ * significantDimension([0,10,20,30]); // -1
+ *
+ * @param {array} values - list of input numbers
```

Comment:
If values is an array of numbers you can also use this:
```suggestion
 * @param {number[]} values - list of input numbers
```